test/tools/libtesteth/boostTest.cpp

```diff
@@ -81,6 +83,17 @@ int main(int argc, const char* argv[])
     std::string const dynamicTestSuiteName = "customTestSuite";
     setDefaultOrCLocale();
 
+    string sMinusTArg;
```

Comment:
This could be done closer to where it's needed - inside `if (result == boost::exit_exception_failure)`.
Also better could be a separate function.